lib/default.nix

```diff
@@ -1,3 +1,4 @@
+{ nixpkgsLib }:
```

Comment:
This breaks existing usage in the overlay.
If you have better suggestion on how to solve this let me know.
I actually I would just get rid of the overlay because it is over-engineering and
makes direct import harder.

Reply:
The overlay doesn't really use lib, so this addition should not matter too much.  Could we have `nixpkgsLib ? import <nixpkgs/lib>` for non-flakes users?

Reply:
> I would just get rid of the overlay because it is over-engineering

We can probably get rid of it. I did a quick search and couldn't find any place where it gets used as overlay anyway.